mp/src/game/shared/momentum/mom_gamemovement.cpp

```diff
@@ -1232,9 +1138,10 @@ bool CMomentumGameMovement::CheckJumpButton()
     }
 
     // Cannot jump while ducked in TF2
-    if (g_pGameModeSystem->IsTF2BasedMode() && (player->GetFlags() & FL_DUCKING))
+    if (g_pGameModeSystem->IsTF2BasedMode())
     {
-        return false;
+        if ((player->m_Local.m_bDucking && (player->GetFlags() & FL_DUCKING)) || player->m_Local.m_flDuckJumpTime > 0.0f)
+            return false;
```

Comment:
Is there a reason this was changed? Because this would only prevent the player from jumping while crouched if they have begun the unducking. Also m_flDuckJumpTime is unused, so there is no reason to check for it.